### PR TITLE
japanese

### PR DESCRIPTION
**Pull request by sabaadmin**
Japanese translation

### CODE REVIEW

**Comment by buiawpkgew1** [comment]
why close?

**Comment by sabaadmin** [comment]
> why close?

I didn't know how to label it.
sorry...

File: src/main/resources/assets/wurst/lang/ja_jp.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    "description.wurst.hack.openwateresp": "「オープンウォーター」で釣りをしているかどうかを示し、オープンウォーターの計算に使用される領域の周りにボックスを描画します。",
+    "description.wurst.hack.overlay": "ブロックをマイニングするたびにNukerアニメーションをレンダリングします。",
+    "description.wurst.hack.panic": "有効なすべてのハックを即座にオフにします。\nこれに注意してください！",
+    "description.wurst.hack.parkour": "ブロックの端に到達したときに自動的にジャンプします。\nパルクールやジャンプ'n'ランに便利です。",
+    "description.wurst.hack.playeresp": "近くのプレイヤーをハイライトします。\n友達のESPボックスは青色で表示されます。",
+    "description.wurst.hack.playerfinder": "雷雨の際に遠くのプレイヤーを見つけます。",
+    "description.wurst.hack.portalgui": "ポータルでGUIを開くことができます。",
+    "description.wurst.hack.potionsaver": "あなたが静止している間、すべてのポーション効果を凍結します。",
+    "description.wurst.hack.prophuntesp": "Prophuntで偽のブロックを表示できます。\nMineplexProphuntのために作られました。 他のサーバーでは動作しない可能性があります。",
+    "description.wurst.hack.protect": "最も近いエンティティを追跡し、他のエンティティから保護するボット。\nUse .protectは、最も近いエンティティではなく特定のエンティティを保護します。",
```
Comment (Alexander01998) [inline src/main/resources/assets/wurst/lang/ja_jp.json:110]: The word "use" is still in English here. Please fix.

File: src/main/resources/assets/wurst/lang/ja_jp.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    "description.wurst.hack.radar": "近くのエンティティの場所を表示します。\n§cred§r - プレイヤー\n§6orange§r - モンスター\n§agreen§r - 動物\n§7gray§r - その他",
+    "description.wurst.hack.rainbowui": "§cM§aa§9k§ce§as §9e§cv§ae§9r§cy§at§9h§ci§an§9g §cc§ao§9l§co§ar§9f§cu§al§9.",
```
Comment (Alexander01998) [inline src/main/resources/assets/wurst/lang/ja_jp.json:112]: This description is still in English. Please translate it to Japanese.

(It says "Makes everything colorful." with each character encoded in a different color. The `§c`, `§a`, `§9` are color codes, the rest is text.)

File: src/main/resources/assets/wurst/lang/ja_jp.json [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    "description.wurst.hack.reach": "より遠くへの到達を可能にします。",
+    "description.wurst.hack.remoteview": "世界を他の誰かとして見ることができます。\n.rvコマンドを使用して、特定のエンティティをターゲットにします。",
+    "description.wurst.hack.safewalk": "ふちから落ちるのを防ぎます。",
+    "description.wurst.hack.scaffoldwalk": "足元にブロックを自動的に配置します。",
+    "description.wurst.hack.search": "特定のブロックをレインボーカラーでハイライトすることで、ブロックの検索をサポートします。",
+    "description.wurst.hack.servercrasher": "1.15.xサーバーをクラッシュさせる可能性のあるアイテムを生成します。\n§oクリエイティブモードのみ。§r",
+    "description.wurst.hack.skinderp": "skinの一部がランダムに切り替わります。",
+    "description.wurst.hack.sneak": "自動的にしゃがみます。",
+    "description.wurst.hack.snowshoe": "パウダースノーの上を歩けるようになります。",
+    "description.wurst.hack.speedhack": "スプリントやジャンプと比べて2.5倍の速さで走ることができます。\n\n§6§l警告:§r NoCheat+のバージョン3.13.2でパッチが当てられました。古いバージョンのNoCheat+をバイパスするだけです。\nType §l/ncp version§r を入力すると、サーバーのNoCheat+バージョンを確認することができます。",
```
Comment (Alexander01998) [inline src/main/resources/assets/wurst/lang/ja_jp.json:122]: The word "type" is still in English here. Please fix.

File: src/main/resources/assets/wurst/lang/ja_jp.json [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    "description.wurst.hack.speednuker": "NoCheat+を回避できないNukerの高速バージョン。",
+    "description.wurst.hack.spider": "蜘蛛のように壁を登れるようになります。",
+    "description.wurst.hack.step": "フルブロックのステップアップを可能にします。",
+    "description.wurst.hack.throw": "1つのアイテムを複数回使用します。雪玉や卵を投げたり、モブを出現させたり、トロッコを置いたりと、非常に大量に使用することができます。\n\nこれが原因でラグが発生し、サーバーがクラッシュすることもあります。",
+    "description.wurst.hack.tillaura": "土や草などを自動的に農地にします。\nKillauraと一緒に使用してはいけません。",
```
Comment (Alexander01998) [inline src/main/resources/assets/wurst/lang/ja_jp.json:127]: Does this say "Do not use with Killaura"? It should say "Do not _confuse_ with Killaura" instead.

**Review by Alexander01998 — CHANGES_REQUESTED** [review]

**Comment by Alexander01998** [comment]
Can someone who natively speaks Japanese please review this and tell me if it's grammatically correct?

**Comment by Alexander01998** [comment]
@sabaadmin Nothing has changed since my last review. Maybe you forgot to upload/push your changes?

**Comment by sabaadmin** [comment]
Fixed the problem

**Comment by sabaadmin** [comment]
> @sabaadmin Nothing has changed since my last review. Maybe you forgot to upload/push your changes?

Sorry for the delay in fixing.
I was too busy to do that.

**Review by Alexander01998 — APPROVED** [review]

**Pull request merged**